src/mobster/oci/__init__.py

```diff
@@ -17,53 +16,11 @@
 import pydantic
 
 from mobster.error import SBOMError
+from mobster.utils import run_async_subprocess
 
 logger = logging.getLogger(__name__)
 
 
-async def run_async_subprocess(
```

Comment:
This is an unrelated change that I started doing because I thought I would need this function outside of the oci module, but I ended up with an alternative. However, this is still a generic enough function that should be in the `utils` module, and I didn't want to throw this out.